addons/account_edi/models/account_edi_format.py

```diff
@@ -622,6 +622,9 @@ def _retrieve_product(self, name=None, default_code=None, barcode=None):
         :param barcode:         The barcode of the product.
         :returns:               A product or an empty recordset if not found.
         '''
+        if name and '\n' in name:
```

Comment:
no need for '\n' in name?

Reply:
This is a forward port, please check the comments on the main PR

Reply:
variable `name` is used to search products. Pretty much every database has product names without `\n`. At the same time, `\n` in the `name` mostly likely comes from mixing product name and sales description. So, I cut everything after `\n`